src/Docs/Session/Retrieve/LivenessResourceResponse.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace Yoti\Docs\Session\Retrieve;
+
+abstract class LivenessResourceResponse extends ResourceResponse
+{
+
+    /**
+     * @return string
+     */
+    abstract public function getLivenessType(): string;
```

Comment:
This method isn't needed as each liveness resource has it's own concrete implementation